d3rlpy/torch_utility.py

```diff
@@ -388,6 +391,19 @@ def reset_optimizer_states(self) -> None:
             if isinstance(v, torch.optim.Optimizer):
                 v.state = collections.defaultdict(dict)
 
+    def get_torch_modules(self) -> List[nn.Module]:
+        torch_modules: List[nn.Module] = []
+        for v in asdict_without_copy(self).values():
```

Comment:
It's nice to return both `key` and `value` for `get_gradients`:
```py
torch_modules = {}
for k, v in asdict_without_copy(self).values():
    if isinstance(v, nn.Module):
                torch_modules[k] = v
return torch_modules
```

Reply:
5b8bb3c